packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md

```diff
@@ -39,32 +55,44 @@ For a more detailed look at the upgrade steps, see the [Plan embedded auth appli
 
-    **Are you embedding our Sign-In Widget?**
+    **Are you embedding the Okta Sign-In Widget?**
 
     * [Upgrade your Okta Sign-In Widget](/docs/guides/oie-upgrade-sign-in-widget/) to the latest version.
 
-    * Test your app to make sure the widget works OK.
+    * Test your app to make sure the widget works.
 
     * Make any necessary [updates to the Sign-In Widget styling](/docs/guides/oie-upgrade-sign-in-widget-styling/) and [i18n properties](/docs/guides/oie-upgrade-sign-in-widget-i18n/).
 
-    **Are you embedding your authentication with our SDKs?**
+    * Check for [deprecated functions or customizations](/docs/guides/oie-upgrade-sign-in-widget/main/#changes-to-widget-configuration-for-identity-engine) that may not be compatible with Identity Engine.
+
+    **Are you embedding your authentication with an Okta SDK?**
 
-    * Add the appropriate Identity Engine SDK to your application code or update to the latest version of the Identity Engine SDK. See [Add the latest Auth SDKs to your applications](/docs/guides/oie-upgrade-add-sdk-to-your-app/nodejs/main/) for detailed steps by language. <!-- (/docs/guides/oie-upgrade-add-sdk-to-your-app/-/main/) -->
+    * Upgrade to the latest version of the Identity Engine SDK. See [Upgrade your app to the Identity Engine SDK](/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/nodejs/main/) for detailed steps by language.
 
-    **Are you moving to an architecture that leverages an Okta SDK?** More detailed information coming soon. <!-- See [Move away from using Classic Authentication APIs to using the appropriate SDK](/docs/guides/) for details on this task. -->
+    * Add the appropriate Identity Engine SDK to your application code. See [Add the latest Auth SDKs to your applications](/docs/guides/oie-upgrade-add-sdk-to-your-app/nodejs/main/) for detailed steps by language.
 
-3. Test your user experience.
+3. Test all of your user experiences.
 
     * **Authentication:** Make sure that your users can sign in and sign out for the workflows that you support.
 
-    * **Self-Service Password Recovery:** Make sure that users can recover their factors with no blockers.
+    * **Self-service & Admin-initiated Password Recovery:** Make sure that users can recover their factors with no impact to the user experience.
+
+    * **New Account Activation:** Make sure that the account onboarding experience works as expected end-to-end to ensure that users have a smooth start.
+
+    * **Self-service registration:** Make sure that users can sign up (including Factor enrollment if you support that) and that there are no interruptions during that process.
+
+    * **Session Management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).
```

Comment:
```suggestion
    * **Session management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).
```

Reply:
I changed each of these to sentence case but if they're actual feature names I'm fine changing them back to capital.